### ISSUE TITLE
BC18: Error during publishing: Reason There is an error in the XML document.

### ISSUE THREAD

**Issue by guidorobben** [microsoft/AL#6572]
Please include the following with each issue:

**1. Describe the bug**
During the publishing of our 18.0 App, I get the following error.

_The request for path /BC/dev/apps?SchemaUpdateMode=forcesync&DependencyPublishingOption=default failed with code UnprocessableEntity. Reason: There is an error in the XML document._

**2. To Reproduce**

Modify in a report the following:

```
UsageCategory = ReportsAndAnalysis;

to 

UsageCategory = ReportsandAnalysis;

```
Change the A to a in the And.

```
Report 11002300 "Checklist Travel Hours"
  {
    DefaultLayout = RDLC;
    RDLCLayout = './Layouts/ChecklistTravelHours.rdlc';

    Caption = 'Checklist Travel Hours';
    UsageCategory = ReportsandAnalysis;`
```

**3. Expected behavior**
I would expect an error in VSC AND a normal error during publishing

**4. Actual behavior**
A clear and concise description of what happened accompanied by images, animations, or a link to a video showing the issue occurring

**5. Versions:**
- AL Language: v7.1.441584
- Compiler: 7.1.6.48380
- Visual Studio Code: 1.55.1
- Business Central: 18.0
- List of Visual Studio Code extensions that you have installed:

**Comment by dtkb** [comment]
related, this used to wait until runtime to crash the web client, so at least now it errors during publishing...

https://github.com/microsoft/AL/issues/6082

**Comment by qutreson** [comment]
This seems like a duplicate of #6513. It should have been fixed now, but it might not have made it to 18.0.

**Comment by dtkb** [comment]
@qutreson Prior to BC18 this didn't give an error during publishing, but instead crashed the web client at runtime. Will that be fixed in BC <= 17?

**Comment by atoader** [comment]
For any version older than 18.0, any request for a backport should go through the standard support channels.

**Comment by qutreson** [comment]
I verified it and this issue has been fixed. It will be available starting from 18.1.
